static_src/components/app_page.jsx

```diff
@@ -0,0 +1,94 @@
+
+import React from 'react';
+
+import AppStore from '../stores/app_store.js';
+
+export default class AppPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.props = props;
+    this.state = {
+      app: {},
+      currentAppGuid: this.props.initialAppGuid
+    };
+    this._onChange = this._onChange.bind(this);
+  }
+
+  componentDidMount() {
+    AppStore.addChangeListener(this._onChange);
+  }
+
+  _onChange() {
+    this.setState({
+      app: AppStore.get(this.state.currentAppGuid) || {}
+    });
+  }
+
+  formatMb(bytes) {
+    return Math.round(bytes / 1000000);
+  }
+
+  render() {
+    let content = <h4 className="test-none_message">No app</h4>;
+
+    if (this.state.app.name) {
```

Comment:
If name isn't there I'm assuming everything else is, might need to be more selective though.